.pre-commit-config.yaml

```diff
@@ -15,15 +15,12 @@
         args:
         - --ignore=W503
     -   id: name-tests-test
-        files: ^test*/.+\.py$
```

Comment:
oh actually, looking more closely at this, it's matching python files in a `tes` `test` or `testttttttttt` directory, heh.